### PR TITLE
Updated to QR code page

### PR DESCRIPTION
**Pull request by ErikCH**
*Issue #, if available:*
#881 

*Description of changes:*
Updated QR code page so users can copy the secret key, instead of using QR code.

https://user-images.githubusercontent.com/65630/148132938-3be26b59-df36-4996-af81-4fa9135ae769.mp4

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: packages/vue/src/components/setup-totp.vue [hunk 1]
```diff
@@ -1,3 +1,98 @@
...
+  qrCodeImageSource: '',
+  isLoading: true,
+});
+let secretKey = ref('');
+let copyTextLabel = ref(translate('COPY'));
+
+function copyText() {
+  navigator.clipboard.writeText(secretKey.value);
+  copyTextLabel.value = translate('COPIED');
+}
```
Comment (ErikCH) [inline packages/vue/src/components/setup-totp.vue:28]: This was the only change, just moved script setup to top.

File: packages/ui/src/theme/css/component/authenticator.scss [hunk 2]
```diff
@@ -139,3 +139,32 @@
...
+[data-amplify-copy] {
+  display: flex;
+  font-size: 0.688rem;
+  gap: 1rem;
+  justify-content: center;
+  align-items: center;
+
+  @media (max-width: 30rem) {
+    word-break: break-all;
+  }
```
Comment (wlee221) [inline packages/ui/src/theme/css/component/authenticator.scss:151]: What do you think about making `[data-amplify-copy]` `hidden` if not on mobile?

Comment (ErikCH) [reply packages/ui/src/theme/css/component/authenticator.scss:151]: I dunno, I was just using a QR code the other day, and I appreciated how the secret key was also displayed. I don't think it has to be just for mobile.

Comment (wlee221) [reply packages/ui/src/theme/css/component/authenticator.scss:151]: That's fair! My only concern is that end-users won't know what to do with that secret key, but customers can use header slots to add directions.

**Review by wlee221 — APPROVED** [review]
🚢

File: packages/ui/src/theme/css/component/authenticator.scss (hunk 2 above)
Comment (ErikCH) [reply packages/ui/src/theme/css/component/authenticator.scss:151]: Yeah, I was debating back and forth if I should add in some instructions on how to add 2FA to Authy, or Google Authenticator, but I didn't want to add more requirements unless a customer wanted too.

And they can add in their own with a slot, so you're right.

**Pull request merged**